src/causalprog/algorithms/moments.py

```diff
@@ -43,4 +43,22 @@ def standard_deviation(
     rng_key: jax.Array,
 ) -> float:
     """Estimate the standard deviation of a graph."""
-    return sample(graph, outcome_node_label, samples, rng_key=rng_key).std()
+    return (
+        moment(2, graph, outcome_node_label, samples, rng_key=rng_key)
+        - moment(1, graph, outcome_node_label, samples, rng_key=rng_key) ** 2
```

Comment:
I'm unsure here if we want to split the rng_key (and so use different samples for the two moments) or use the same key so that the two moments use the same sample data

Reply:
This seems like a sensible solution that we can revisit later if needed